index.html

```diff
@@ -57,5 +57,183 @@ <h1 class="self">
 
   </section> 
 
+<!--Work section title-->
+
+  <div class="showcase-title">
+    <div class="title-box">
+      <h2>My Recent Works</h2>
+    </div>
+    <div class="separator"></div>
+  </div> 
+
+<!--Work Section-->
+
+  <section id="worksection">
+
+    <div class="card item1">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item2">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item3">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item4">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item5">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="card item6">
+      <div class="card-info">
+        <h3>Multi-Post Stories Gain+Glory</h3>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="btnbar">
+          <p class="btn"><a href="#">See project</a></p>
+        </div>
+      </div>
+    </div>    
+
+  </section>
+
+<!--About me Section-->
+
+  <section id="aboutme">
+
+    <div class="aboutwrapper">
+
+      <img src="./image/Mask Group.png" alt="design element" class="top-img">
+      <img src="./image/Group.png" alt="design element" class="bottom-img">
+
```

Comment:
- [ ] It is important to layout the images for the about section using background images in css rather than making use of semantic img tag. The two highlighted images are meant to serve as background images for this section so please kindly use the provided css property to present both images. :+1: